src/redux/actionTypes/actionTypes.js

```diff
@@ -0,0 +1,2 @@
+export const GREETING = 'GREETING';
```

Comment:
Change this file path to src/actions/actionTypes.js, there is no need for the redux folder

Reply:
Done